docs/populating-configuration.md

```diff
@@ -18,17 +18,62 @@ to test things out.
 
 >NOTE: You could pass dummy values to make sure that the secrets files are created.
 
-## Setting up OCM token
+## Setting up OCM tokens
 
-[OpenShift Cluster Managament](https://api.openshift.com/) is utilised for the management of Data plane clusters. 
-To be able to talk to this service, retrieve your ocm-offline-token from https://qaprodauth.cloud.redhat.com/openshift/token 
-and save and run the following command
+[OpenShift Cluster Management](https://api.openshift.com/) in Fleet Manager is
+used for mainly two purposes:
+* By Fleet Manager, to perform management of Data Plane clusters
+* By the users, to interact with the Fleet Manager API: OpenShift Cluster
+  Management (OCM) tokens are used as an Authentication mechanism to interact
+  with the Fleet Manager API
```

Comment:
`ocm token` is just a convenient way of getting a jwt token from sso.redhat.com, the default sso used for authentication: https://github.com/bf2fc6cc711aee1a0c2a/fleet-manager-go-template/blob/b705f061d0186e291741eedbeca8a4077c93be9b/pkg/server/server_config.go#L27-L29

so technically, a user does not need `ocm token` but rather a jwt token from sso.redhat.com. The ocm cli has just all of this connection mechanism built that's why we rely on it.

Reply:
Reworded a little bit